src/app/lib/constants/styles.js

```diff
@@ -39,3 +39,26 @@ export const GEL_SPACING_QUAD = `2rem`;
 export const smartPhoneScreenWidthMin = 20; // 320px
 export const smartPhoneScreenWidthMax = 37.4375; // 599px
 export const laptopScreenWidthMin = 37.5; // 600px
+
+/* 
+    The following are breakpoints from GEL Grid
+    Link to relevant GEL docs: https://www.bbc.co.uk/gel/guidelines/grid#grid-sizes
+    The only exception is that we have split out group 1 into 0 and 1 
+*/
+
+export const group0ScreenWidthMin = `0em`; // 0px
```

Comment:
In a separate PR, I'll address the other variables above e.g. `laptopScreenWidthMin` to move to this consistent naming convention that is also used by the UX team.